src/bin/cli/ideascale/mod.rs

```diff
@@ -0,0 +1,138 @@
+use catalyst_toolbox::ideascale::{
+    build_challenges, build_fund, build_proposals, fetch_all, Error,
+};
+
+use structopt::StructOpt;
+
+use serde::Serialize;
+use std::fmt::{Display, Formatter};
+use std::path::{Path, PathBuf};
+use std::str::FromStr;
+
+#[derive(Debug)]
+pub enum VoteType {
+    Public,
+    Private,
+}
+
+#[derive(Debug, StructOpt)]
+pub enum Ideascale {
+    Import(Import),
+}
+
+#[derive(Debug, StructOpt)]
+#[structopt(rename_all = "kebab")]
+pub struct Import {
+    /// Fund number id
+    #[structopt(long)]
+    fund: usize,
+
+    /// Fund goal explanation
+    #[structopt(long)]
+    fund_goal: String,
+
+    /// ideascale API token
+    #[structopt(long)]
+    api_token: String,
```

Comment:
It's somewhat better security to pass the token via an environment variable rather than expose it on the command line. In CI environments this could come from a secret, be screened in logs etc.

Reply:
Actually yes, let me let it load from both, since it is quite easy to do.